### ISSUE TITLE
form element's :focus

### ISSUE THREAD

**Issue by anth12** [FatMedia/FatKit#17]
For accessibility checkbox & radio buttons should have a default focus highlight. May also be applicable to buttons?

**Comment by ahallicks** [comment]
Added focus to advanced form controls (default forms controls aren't styled by FatKit so will inherit browser styles)
